api/lib/opentelemetry/logger.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+require 'logger'
+
+module OpenTelemetry
+  class << self
+    attr_accessor :logger
```

Comment:
would it make more sense to put it to https://github.com/open-telemetry/opentelemetry-ruby/blob/2a22a85e09a2c4823a804d71481c3e4cc606f5fd/api/lib/opentelemetry.rb which already includes similar things and it's on top level also ?

Reply:
we could for sure.  I started this thinking we would subclass `Logger` but I didn't see any reason at the moment. 

any opinions @fbogsany ?

Reply:
Nope, no opinions.

Reply:
lets see what everyone thinks.  I'm ok either way.

Reply:
I moved it in